bigframes/ml/llm.py

```diff
@@ -694,19 +694,19 @@ def _from_bq(
         model._bqml_model = core.BqmlModel(session, bq_model)
         return model
 
-    def predict(self, X: Union[bpd.DataFrame, bpd.Series]) -> bpd.DataFrame:
+    def predict(self, X: utils.ArrayType) -> bpd.DataFrame:
         """Predict the result from input DataFrame.
 
         Args:
-            X (bigframes.dataframe.DataFrame or bigframes.series.Series):
+            X (bigframes.dataframe.DataFrame or bigframes.series.Series or pandas.core.frame.DataFrame or pandas.core.series.Series):
```

Comment:
I worry that fully enumerating the accepted types will be too much once we further extend

Reply:
I agree with you. The Google style prefers type hints over type documents, and it makes more sense. Here I'm just keeping the style consistent.